src/routes/things.js

```diff
@@ -47,8 +57,104 @@ let router = express.Router();
  * }
  **/
 
-router.get('/Things', (req, res) => {
-  res.status(200).send();
+const resource = 'Things';
+
+router.get('/', (req, res) => {
+  db().then((models) => {
+    if (req.params && req.params[0]) {
+      models.Things.findById(req.params[0], {
+        attributes: {
+          exclude: excludedFields
+        }
+      }).then(thing => {
+        res.status(200).json(response.generate(thing));
```

Comment:
If the given id is not valid, we will get `thing === null`, we need to return a 404 in that case.